apps/wallet-dashboard/components/Dialogs/Staking/StakeDialog.tsx

```diff
@@ -78,114 +66,69 @@ export function StakeDialog({
                 maxStakableTimelockedAmount ?? coinBalance,
                 coinSymbol,
                 coinDecimals,
-                view === StakeDialogView.Unstake,
+                false,
```

Comment:
no, it's because we moved the Unstake view to a "Unstake" Dialog, it's not a view of the Stake Dialog anymore, so now we have `UnstakeDialog` with the `UnstakeView` and the `StakeDialog` with other views, so this conditional in this dialog is always false